src/chef-server-ctl/lib/chef_server_ctl/helpers/du.rb

```diff
@@ -0,0 +1,22 @@
+require 'mixlib/shellout'
+module Du
+  # Calculate the disk space used by the given path. Requires that
+  # `du` is in our PATH.
+  #
+  # @param path [String] Path to a directory on disk
+  # @return [Integer] KB used by directory on disk
+  #
+  def self.du(path)
+    command = Mixlib::ShellOut.new("du -sk #{path}")
+    command.run_command
+    if command.status.success?
+      command.stdout.split("\t").first.to_i
+    else
+      puts "du -sk #{path} failed with exit status: #{command.exitstatus}"
+      puts "du stderr: #{command.stderr}"
+      raise 'DuFailedException', command.stderr(FFI.errno)
```

Comment:
Does command.stderr accept an argument?

Reply:
Looking like it is not accepting any argument, Do we have any docs for referring command module functions arity.I have verified this by running some smaple tests.